src/common/tests/utest_common.c

```diff
@@ -286,3 +294,105 @@ utest_utx2uma(struct utest_context *utx)
 {
 	return &utx->uc_uma;
 }
+
+int
+utest_get_scm_used_space(struct utest_context *utx,
+						daos_size_t *used_space)
```

Comment:
Saw that Jeff already commented on alignment and you fixed. Was this one missed/reverted?

Reply:
I fixed it... It looks good under my notepad++... Looks like editor setup issue in this area.